corehq/apps/translations/app_translations/upload_module.py

```diff
@@ -140,6 +144,20 @@ def _update_case_search_labels(self, rows):
                     self.msgs.append((messages.error, message))
                     continue
                 self._update_translation(display_row, prop.label)
+        else:
+            message = _(
```

Comment:
should we have this same message for hints for consistency?

Reply:
I think it'll usually be the case that not all case properties will have hints (whereas they're all required to have a display text) so having a similar error for hints I think sends the wrong message